src/lib/services/pat-service.ts

```diff
@@ -55,6 +55,20 @@ export default class PatService {
         return this.patStore.deleteForUser(id, userId);
     }
 
+    async validatePat({ description, expiresAt }: IPat): Promise<void> {
+        if (!description) {
+            throw new BadDataError('PAT description cannot be empty');
+        }
+
+        if (new Date(expiresAt) < new Date()) {
+            throw new Error('The expiry date should be in future.');
```

Comment:
I know you've just moved this to here and it's not really part of this PR but this error type causes the API to freak out, would it cause trouble if we change this to a BadDataError instead? 

```suggestion
            throw new BadDataError('The expiry date should be in future.');
```